.gitattributes

```diff
@@ -0,0 +1,3 @@
+zz_generated* linguist-generated=true
+pkg/client linguist-generated=true
+config/crds/*.yaml linguist-generated=true
```

Comment:
Will changes in these files still show up in the diff? We might still want to know that they changed without knowing what exactly has changed.

Reply:
They will, just collapsed by default

Reply:
Example: https://github.com/kubewharf/kelemetry/pull/109/files#diff-be72556c262b17baaa99cc522dd96991b063b2a9e5ee3af66fc20feb8768d00f
<img width="869" alt="image" src="https://github.com/kubewharf/kubeadmiral/assets/19623715/b15cc99a-b468-473e-a117-05fda641b59d">

Reply:
In addition to collapsing in diff, another effect is that it will exclude generated files from code line stats, but idk if this will affect existing contributor line count